### ISSUE TITLE
can't install all necessary libraries in order to using OpenAI along with Autolabel 

### ISSUE THREAD

**Issue by YuranZhang2000** [refuel-ai/autolabel#620]
pip install 'refuel-autolabel[openai]'
Note: you may need to restart the kernel to use updated packages.
ERROR: Invalid requirement: "'refuel-autolabel[openai]'"

!pip3 install 'refuel-autolabel[openai]'
ERROR: Invalid requirement: "'refuel-autolabel[openai]'"

**Comment by shril** [comment]
Looks like there is some problem with you local pip installation.
Can you specify what version of python, pip, package manager you are using?

Meanwhile, lmk if this stackoverflow answer helps - [Link](https://stackoverflow.com/a/35625489/6371154).

**Comment by yingyin-chan** [comment]
Hello, have you solved it? I have also encountered the same problem! Looking forward to your reply, thank you!

**Comment by YuranZhang2000** [comment]
No, I did not solve it. But it was a few months before, and I never tried to install it again after that.
&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "refuel-ai/autolabel"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年5月18日(星期六) 晚上10:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [refuel-ai/autolabel] can't install all necessary libraries in order to using OpenAI along with Autolabel  (Issue #620)





 
Hello, have you solved it? I have also encountered the same problem! Looking forward to your reply, thank you!
